Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -43,6 +43,8 @@ class XMLHttpRequestBase {
   readyState: number;
   responseHeaders: ?Object;
   responseText: ?string;
+  response: ?Object;
+  responseType: ?string;
```

Comment:
As per https://xhr.spec.whatwg.org/#the-responsetype-attribute, values are: the empty string (default), "arraybuffer", "blob", "document", "json", and "text". Hence I opted for string type.